airflow/bin/cli.py

```diff
@@ -857,7 +857,8 @@ def clear(args):
                 task_regex=args.task_regex,
                 include_downstream=args.downstream,
                 include_upstream=args.upstream)
-
+    if args.yes:
+        args.no_confirm = args.yes
```

Comment:
Do we need a new argument for it?

We can just update L868 to:

```diff
- confirm_prompt=not args.no_confirm,
+ confirm_prompt=not args.yes
```

Reply:
I saw that no_confirm has been around for the command for a while, that made me not to remove it entirely. I will remove it now.